draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -102,19 +102,19 @@ specification. {{sctp-Crypto-chunk-layering}} illustrates the CRYPTO
 chunk layering in regard to SCTP and the Upper Layer Protocol (ULP).
 
 ~~~~~~~~~~~ aasvg
++---------------+ +----------------------------------+
+|               | |           Protection Engine      |
+|      ULP      | +----------------+                 |
+|               | | Key Management |                 |
++---------------+-+---+----+-------+                 |
+|                     |    |                         |
+| SCTP Chunks Handler | +-- SCTP Unprotected Payload |
+|                     |/   |                         |
++---------------------+    +--------------------+    |
+|        CRYPTO       |    |  DATA Management   |    |
+|        Chunk        |<-->|                    |    |
+|       Handler       |    +--------------------+----+
```

Comment:
More changes are needed a couple of paragraphs below this figure as it describes the protection engine initilization as sending crypto frames, not DATA chunks with a special PPID.